artifacts/agent/namespace.yaml

```diff
@@ -2,3 +2,9 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: karmada-system
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: karmada-cluster
```

Comment:
@Garrybest We are going to create the `karmada-cluster` namespace for secretes used by `karmada-agent`.

Reply:
Got. We may need to pay attention to version transition.